Formula/usbutils.rb

```diff
@@ -0,0 +1,282 @@
+class Usbutils < Formula
+  desc "Provides 'lsusb' tool for getting detailed info about USB devices"
+  # Homepage for multiple Linux USB tools, 'usbutils' is one of them.
+  homepage "http://www.linux-usb.org/"
+  url "https://github.com/gregkh/usbutils.git",
```

Comment:
Shouldn't the official upstream release be used instead of a git snapshot?

Reply:
@SMillerDev can you help with this?

Reply:
See https://github.com/Homebrew/homebrew-core/pull/89341#discussion_r750726447. autoreconf wouldn't be needed.

Reply:
`configure` caches `libudev` requirement without `autoreconf`:
```
configure: error: Package requirements (libudev >= 196) were not met:

No package 'libudev' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.
```

Reply:
Yes, you're right of course, the autoreconf is still needed. In one of my patches configure.ac is changed to make udev optional. For this to be reflected in configure, autoreconf must be run to regenerate it.